libs/wingsdk/src/shared-aws/topic.inflight.ts

```diff
@@ -1,17 +1,63 @@
-import { SNSClient, PublishCommand } from "@aws-sdk/client-sns";
+import {
+  SNSClient,
+  PublishBatchCommand,
+  PublishBatchRequestEntry,
+  InvalidBatchEntryIdException,
+} from "@aws-sdk/client-sns";
 import { ITopicClient } from "../cloud";
+import { Util } from "../util/util";
+
+const CHUNK_SIZE = 10;
```

Comment:
please add a comment saying that it's aws current limit :)

Reply:
Sure, I forgot to do that.